examples/trials/NAS/README.md

```diff
@@ -1,8 +1,8 @@
- **Run Neural Network Architecture Search in NNI**	
- ===	
- 
-Now we have an NAS example [NNI-NAS-Example](https://github.com/Crysple/NNI-NAS-Example) run in NNI using NAS interface from our contributors.	
- 
-Thanks our lovely contributors. 	
- 
-And welcome more and more people to join us!
+ **Run Neural Network Architecture Search in NNI**	
+ ===	
```

Comment:
I suggest that we use do not use "NAS" as the folder name, because network morphism is a NAS method, too.

Actually, it based on ENAS controller, so...

Reply:
good suggestions, I also do not think this is a good name.

Reply:
renamed to nas_cifar10